label_studio/organizations/urls.py

```diff
@@ -3,36 +3,48 @@
 from django.urls import include, path
 from organizations import api, views
 
-app_name = 'organizations'
+app_name = "organizations"
```

Comment:
as I can see the rest of python code uses single quotes, so that can mess with their linting

Reply:
fixed